docs/changelog.md

```diff
@@ -1,3 +1,139 @@
+- Removed unused NEMO simulator compatibility (#3035)  
+- Removed checkpoint feature (`h.checkpoint()`), superseded by SaveState (#3092)  
+- Removed legacy Random.MLCG and Random.ACG random number generators (#3189, #3190)  
+- Removed support for mod2c transpiler for CoreNEURON (#2247)  
+- Removed unused NMODL constructs: PUTQ, GETQ, RESET, MATCH, TERMINAL, SECTION,  
+  IFERROR, MODEL_LEVEL, PLOT, SENS, etc. (#1956, #1974, #1975, #2001, #2004, #2005)  
+- Removed `__MWERKS__` (CodeWarrior compiler) related code (#2655)  
+- Removed obsolete uxnrnbbs code (#2203)  
+- Removed old tqueue implementations (#2225, #2740)  
+- Removed unused RxD code_matrix_to_rxd_sparse (#3547)
+- Py_NoSiteFlag is deprecated for site_import (#3286)
+
+### Bug Fixes
+
+- Fix segfault from unref hoc_obj, see #1857 (#1917)  
+- Fix bug in rxd reactions involving rxd parameters (#1933)  
+- Fix CoreNEURON bug when checkpointing VecPlay instances (#2148)  
+- Fix nocmodl bug with unused STATE + COMPARTMENT variable in KINETIC block (#2210)  
+- Fix for dynamic ECS diffusion characteristics (#2229)  
+- Various fixes for compatibility with newer NVHPC compiler releases (#2239, #2591)  
+- Fix Windows and Mac crash on multiline HOC statements input from terminal (#2258)  
+- Fix for Vector.max_ind (#2251)  
+- Fix for detecting ION channel type in LoadBalance (#2310)  
+- Fix for RangeVarPlot adding extra point at section ends when using plotly (#2410)  
+- Fixed mod file array variable assignment (seg.mech.array[i] = x) (#2504)  
+- Various fixes in RxD features (#2593, #2588)  
+- Fix CoreNEURON bug causing in-memory model copy with file mode (#2767)  
+- Fix CoreNEURON bug when using array variables in MOD files (#2779)  
+- Fix lexer for ONTOLOGY parsing (#3091)  
+- Fix issue while using Anaconda Python on MacOS (#3088)
+- Fix ParallelContext bbs memory leaks (#3563)
+- Fix RxD indexing bug? (#3337)
+- Fix handling of `hh.mod` when using CoreNEURON (#3301)
+- On h.quit(), terminal settings are same as when neuron.hoc was imported. (#3259)
+- Int replaced by floor to accept negative coordinates. (#3264)
+- Fix thread sanitizer leak when launching Python 3.13 and using from
+- Fix sanitizer leak involving `neuron import h, gui`. (#3243)
+- Fix leaks in `get_endian_character`. (#3257)
+- Fix leak in `pysec_children` and `pysec_subtree`. (#3255)
+- Python 3.13.1 broke [s for s in sl] where sl is a SectionList. (#3276)
+
+
+### Other Improvements and Changes
+
+- Allow `segment.mechanism.func(args)` to call a FUNCTION or PROCEDURE (#2475)  
+- Simplify INDEPENDENT statement as only `t` variable is accepted (#2013)  
+- Fix various memory leaks and related improvements (#3255, #3257, #3243, #2456)  
+- Support for reading reporting related information with in-memory mode (#2555)  
+- NMODL language documentation update (#2152, #2771, #2885)  
+- Documentation improvements including transfer from nrn.yale.edu to nrn.readthedocs.io (#1901, #2922, #2971, #3106, #3187)  
```

Comment:
Would maybe be useful to add `https://` in front of the URLs to make them clickable when displayed.

Reply:
Can you show me an example of what you mean, eg. in line 103.

Reply:
<img width="322" height="88" alt="Screenshot 2025-09-29 at 13 26 45" src="https://github.com/user-attachments/assets/eff3a2de-a2a7-4d89-a81b-6b30cf9abeef" />

As shown above, the links are not clickable in the rendered version (i.e. there's no `<a href=...` HTML element). If you add `https://` in front, they should become clickable because Markdown renders it as such.

- nrn.readthedocs.io

vs.

- https://nrn.readthedocs.io

Reply:
the two urls are now clickable